docs/lookout-sdk.md

```diff
@@ -51,36 +51,45 @@ If we look at this example history of a given local repository stored at `/somew
 $ git log --pretty=oneline --graph
 
 *   d036524c463227524f4bbd7b207fb87bb8b89ee3 (HEAD -> master) Merge pull request #3
-|\
+|\  
 | * 045a24828327ac35a28186f9b9b437adc3f7b7a3 (branch-b) message
 | * 804cbd94869cb173494ce1de410f2b48674bc772 message
-|/
-*   9294ddb13cc7c8acd2db480c9e5c1396cd85e50a (branch-a) message
-*   7f2ee64cd0a5891900cc368ae35e60a61c262060 message
+|/  
+*   9294ddb13cc7c8acd2db480c9e5c1396cd85e50a Merge pull request #2
+|\  
+| * 355f001d719bd0368c0469acd1a46298a80bacc0 (branch-a) message
+| * 7f2ee64cd0a5891900cc368ae35e60a61c262060 message
+|/  
+*   fa97fa19e5c9b3482e5f88e264fb62b1e7fc6d8f Merge pull request #1
+*
 *
 ...
 ```
 
-And considering we have an analyzer listening at `ipv4://localhost:9999`
+If your current directory is this repository's path, and your analyzer is listening on the default port `9930`, you can run:
+```shell
+$ lookout-sdk review
+```
+
+Doing so, `lookout-sdk` will:
+
+1. start a gRPC **source{d} Lookout DataService** endpoint backed by the repository stored at your current directory.
+1. send a gRPC `NotifyReviewEvent` call to your analyzer listening on `ipv4://localhost:9930`. The `ReviewEvent` argument will contain a `commit_revision` field made of:
+    * `base` pointing to `9294ddb...` (`HEAD^`)
```

Comment:
Since we're using defaults here, I'd use change the order
```suggestion
    * `base` pointing to `HEAD^` (`9294ddb...`)
```

Reply:
Thanks. Done, rebased and squashed.